packages/cactus-core-api/src/main/rust/generated/proto-rs/common.access_control.rs

```diff
@@ -0,0 +1,21 @@
+/// AccessControlPolicy specifies a set of data that can be accessed by some
```

Comment:
This file doesn't seem to be a relevant dependency. Why is it getting built?

Reply:
@VRamakrishna My bad once again, it was a leftover from the various different tries I executed while I was trying to get the compilation to work in the initial phase of the work.